DESCRIPTION

```diff
@@ -2,8 +2,8 @@ Encoding: UTF-8
 Package: goldfish
 Type: Package
 Title: Goldfish -- Statistical network models for dynamic network data
-Version: 1.4.9
-Date: 2020-07-14
+Version: 1.5.0
+Date: 2021-04-28
```

Comment:
@auzaheta you've indicated in the PR description and in commit comments that this is a #major bump, but here in the DESCRIPTION you've incremented only #minor. (Remember semantic versioning goes major.minor.patch...). If we follow [semver](https://semver.org) then major changes are reserved for when we really break/make incompatible changes to the core API.

Reply:
I guess you'll also need to update the date...

Reply:
I'll update the date. You're right I misunderstood the use of #major and #minor. I've should be using #minor instead.

Reply:
Sorry, my bad, I put the date when we merged to the develop not the main!

Reply:
Can we update it again now then?